src/Solnet.Programs/Stake/StakeProgram.cs

```diff
@@ -0,0 +1,474 @@
+﻿using Solnet.Programs.Utilities;
+using Solnet.Rpc.Models;
+using Solnet.Wallet;
+using System;
+using System.Collections.Generic;
+using static Solnet.Programs.Models.Stake.State;
+
+namespace Solnet.Programs
+{
+    /// <summary>
+    /// Implements the Stake Program methods.
+    /// <remarks>
+    /// For more information see:
+    /// LINK HERE
```

Comment:
Either add the links or remove the remarks. I'm assuming this would be links for the StakeProgram rust repos?